libs/agno/agno/workflow/workflow.py

```diff
@@ -447,6 +466,8 @@ def update_run_method(self):
                 object.__setattr__(self, "run", self.run_workflow.__get__(self))
             elif run_type == "async":
                 object.__setattr__(self, "arun", self.arun_workflow.__get__(self))
```

Comment:
Should this one be "coroutine" or "async_coroutine"? Just "async" is ambiguous now